### DISCUSSION TITLE
Auto formatting

### DISCUSSION THREAD

**Discussion by SSmale** [jdneo/vscode-checkstyle#357] [General]
I am looking for a way to get my VSCode setup to auto fix the check style issues (as best as it can) so that there can be a check on build and I can be confident that I will have met the standard, Is there a guide to do this?

**Comment by jdneo** [comment]
Thank you @SSmale.

Looks like something needs to be done to integrate the extension with

```
"editor.codeActionsOnSave": {
    "source.fixAll": "xxx"
}
```

I'll create an issue for this: https://github.com/jdneo/vscode-checkstyle/issues/358
